inference-engine/src/vpu/graph_transformer/src/stages/ceiling.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
Nit: I think `2020` only should be here.

Reply:
Fixed